app/controllers/api/v1/inventories_controller.rb

```diff
@@ -48,34 +48,27 @@ def destroy
 
       private
 
-      def load_team
-        @team = Team.find(params.require(:team_id))
-        render jsonapi: {}, status: :forbidden unless can_read_team?(@team)
-      end
-
       def load_inventory
```

Comment:
Correct me if I'm wrong, but I have a possible suggestion: when we're loading various data objects in API controllers (e.g. this `load_inventory` right here), the only difference from the `BaseController.load_inventory` is that there, the parameter we're querying is `:inventory_id` in `BaseController`, but here it's `:id`.

Can we just update the `load_*` methods in `BaseController` to have a parameter key (which would default to e.g. `:inventory_id`), e.g.:

```ruby
def load_inventory(key = :inventory_id)
  @inventory = @team.repositories.find(params.require(key))
end
```

And then within dedicated controllers like this one, we can still reuse that `BaseController` method, we just pass a parameter to it, e.g.:
```ruby
before_action only: %i(show update destroy) { load_inventory(:id) }
```

This, of course, only works, if the methods are completely the same, with the exception of the parameter. If that's not the case, ignore this suggestion.

Reply:
Good point! Will implement it.